src/cloud-api-adaptor/libvirt/config_libvirt.sh

```diff
@@ -140,3 +138,8 @@ case $TEST_E2E_SECURE_COMMS in
     echo "SECURE_COMMS=\"false\"" >> libvirt.properties
     ;;
 esac
+
+if [[ "${OS_DISTRO}" == "ubuntu" ]] && [[ "${CI}" != "true" ]]; then
```

Comment:
Hi @stevenhorsman ,

One last fix, use `${CI:-}` to make `set -o nounset` happy.

Reply:
It is interesting that the GHA shellcheck didn't pick up on that, maybe that's something we need to be aware of

Reply:
hmmm... are we shell-checking the scripts on workflows or all scripts?